products/views.py

```diff
@@ -1,3 +1,49 @@
-from django.shortcuts import render
+import json, jwt
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+
+from products.models  import Product, ProductSize
+from users.utils      import login_decorator
+
+class ListView(View):
```

Comment:
```suggestion
class ProductListView(View):
```
or
```suggestion
class ProductsView(View):
```
해당 View가 어떤 자원에 대한 View인지를 명확히 나타낼 수 있는 class 명으로 바꿔주세요.

Reply:
넵! 확인했습니다~